sample/src/main/java/com/canhub/cropper/sample/extend_activity/app/SExtendActivityJava.java

```diff
@@ -0,0 +1,34 @@
+package com.canhub.cropper.sample.extend_activity.app;
```

Comment:
Please add a new package `camera_java` so we have `com.canhub.cropper.sample.camera_java.app`

Reply:
Still open